internal/controller/config/crd/bases/agent-install.openshift.io_agents.yaml

```diff
@@ -62,6 +62,9 @@ spec:
                 type: object
               hostname:
                 type: string
+              ignitionConfigOverride:
+                description: Json formatted string containing the user overrides for the host's ignition config
```

Comment:
Isn't this generated from the object above? In that case, shouldn't this be added as comment on the struct property?

Reply:
Done, Thank you.